lib/qb_integration/services/purchase_order.rb

```diff
@@ -0,0 +1,45 @@
+module QBIntegration
+  module Service
+    class PurchaseOrder < Base
+      attr_reader :payload, :purchase_order, :line_service, :account_service, :vendor_service
+
+      def initialize(config, payload)
+        @payload = payload
+        @purchase_order = payload[:purchase_order]
+        @line_service = Line.new config, payload
+        @account_service = Account.new config
+        @vendor_service = Vendor.new config
+        super("PurchaseOrder", config)
+      end
+
+      def create
+        new_purchase_order = create_model
+        build new_purchase_order
+        quickbooks.create new_purchase_order
+      end
+
+      private
+
+      def build(new_purchase_order)
+        new_purchase_order.doc_number = purchase_order["id"]
+        new_purchase_order.vendor_address = Address.build purchase_order["supplier_address"]
+        new_purchase_order.ship_address = Address.build purchase_order["shipping_address"]
+
+        if config["quickbooks_vendor_name"].present?
```

Comment:
Let's move these to the same conditional idea we discussed for add_product and update_product.
`purchase_order["quickbooks_vendor_name"] || config["quickbooks_vendor_name"]`

Same for `quickbooks_account_name`